### DISCUSSION TITLE
[Web] Which features are available?

### DISCUSSION THREAD

**Discussion by ollyde** [flutter-stripe/flutter_stripe#785] [Q&A]
From the docs

```
Web (Experimental)
Now you can use Stripe with Flutter web! Notice right now it is highly experimental and only a subset of features is implemented.
```

But which features are available?

**Comment by ollyde** [comment]
We'd want to be a customer of stripe. One huge issue with iOS/Android apps is rip off IAP duopoloy which we are avoiding with web payments. There doesn't seem to be any library in Flutter that supports web payments.

My question is
- Do you support subscriptions?
- Do you support web-payments with credit/debit cards?
- Can we process bank challenge web screens?

I don't mind if they are highly experimental. I just need to know what basis is supported.

**Comment by jonasbark** [comment]
You can see the un-/supported operations here:
https://github.com/flutter-stripe/flutter_stripe/blob/main/packages/stripe_web/lib/src/web_stripe.dart

Generally, I recommend using stripe_checkout instead for Web payments.

**Reply by ollyde** [reply]
@jonasbark great thanks for the tip

**Reply by ollyde** [reply]
@jonasbark might be good to place this link in the readme for others?
